www/activities/settings/js/settings.js

```diff
@@ -613,6 +613,7 @@ app.Settings = {
       diary: {
         "meal-names": ["Breakfast", "Lunch", "Dinner", "Snacks", "", "", ""],
         timestamps: false,
+        "show-brand": false,
```

Comment:
The property name here needs to match the one used elsewhere in the code. I suggest changing it to "show-brands".

Also, I think this setting should be true by default because most users will want to see the brand.

Finally, you also need to add this setting to the migrateSettings function (further below in this file) and set it to true there to make sure that users who already have the app installed and upgrade to the new version don't suddenly have their brands disappear from the diary.

Reply:
Oh, sorry again, missed this. 
Will adjust my branch for the default value, and the migrateSettings.